src/back/controllers/auth/auth.service.spec.js

```diff
@@ -0,0 +1,159 @@
+'use strict';
+
+const proxyquire = require('proxyquire'),
+    expect = require('chai').expect;
+
+describe('auth.service', () => {
+    let sut,
+        jsonwebtoken,
+        expressJWT,
+        envValues,
+        userModel,
+        userInstance;
+
+    const expressJWTResult = function expressJWTResult() {
+        return () => {
+        };
+    };
+
+    const token = 'token';
+    beforeEach(() => {
+        jsonwebtoken = {
+            sign: env.spy(() => token)
+        };
+        expressJWT = env.spy(expressJWTResult);
+        envValues = {
+            SESSION_SECRET: 'testSecret'
+        };
+        userInstance = {
+            save: env.stub()
+        };
+        userModel = env.spy(() => (userInstance));
+        userModel.findOne = env.stub();
+        sut = proxyquire('./auth.service', {
+            jsonwebtoken,
+            'express-jwt': expressJWT,
+            '../../../../env': envValues
+        });
+    });
+
+    it('should export validateJwt middleware', () => {
+        expect(sut.validateJwt).to.be.an.instanceof(Function);
```

Comment:
Do we really need to test it is a function? It is responsibility of `'express-jwt'`package. I believe it is better to make a test like: `expect(sut.validateJwt.name).to.be('expressJWTResult')` or `expect(sut.validateJwt).to.be(expressJWTResult)`